wrapper/hosts.py

```diff
@@ -595,77 +602,289 @@ def sdk_connection(self, data):
                 username=str(username),
                 password=str(data['rhv_password']),
                 log=logging.getLogger(),
+                debug=data.get('rhv_debug', False),
```

Comment:
We should mention that in documentation someday. Possibly add "Debugging" section with things that can change without warning.